benchmark/sirun/startup/meta.json

```diff
@@ -13,6 +13,18 @@
       "env": {
         "USE_TRACER": "1"
       }
+    },
+    "control-everything": {
```

Comment:
What is the purpose of this benchmark? It seems to want to test isolating importing the tracer, but it doesn't achieve that since the tracer is both imported _and_ initialized.

Reply:
`control-everything` is a baseline case for `with-tracer-everything`. The idea is to check the impact of our loader code when there are a lot of dependencies.

Reply:
Still not sure I understand the goal. Requiring all dependencies as-is without using them in any way is not something the tracer would ever do. It also doesn't load any plugin in a realistic way.